When fork for ipa im Getting error on Building Tweak for Sideloading

### Describe the bug

```yaml
i don´t know why i am getting the error i used a decrypted ipa and set read write permissions
```


### Steps to reproduce

```yaml
1. fork
2. Build with decrypted ipa
3. error
```


### Crashlog

[logs_23178718541.zip](https://github.com/dayanch96/YTMusicUltimate/files/15128398/logs_23178718541.zip)


### Post Requisites

- [X] I have checked that a similar issue has not been reported before.
- [X] Verified the relevance of the project and the version you are using.
- [X] Provided a detailed description of the problem and have filled in all required fields.
- [X] I have read the 'IPA Building Troubleshooting' section (Only for issues related to building the IPA file).
- [ ] I did not answer truthfully to **ALL** the above checkboxes.
- [X] By submitting this issue, **I confirm that I have filled in all the necessary fields and complied with the guidelines. Failure to comply may result in the issue being closed without discussion, and repeated violations may lead to restrict account from further participation.**

@dayanch96 
I think @DRECKINTERNET is talking the the bug below
```log
==> Compiling Source/YTMTab.x (arm64)…
In file included from Source/BackgroundPlayback.xm:13:
==> Compiling Source/BackgroundPlayback.xm (arm64)…
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:45:1: error: import of C++ module 'MachO.nlist' appears within extern "C" language linkage specification [-Wmodule-import-in-extern-c]
#include <mach-o/nlist.h>
^
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:[43](https://github.com/huitzlopochtli/YTMusicUltimate/actions/runs/8848868536/job/24299535693#step:12:44):1: note: extern "C" language linkage specification begins here
extern "C" {
^
1 error generated.
make[3]: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/instance/rules.mk:387: /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/BackgroundPlayback.xm.10b1eca7.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from Source/PremiumStatus.xm:26:
==> Compiling Source/PremiumStatus.xm (arm64)…
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:[45](https://github.com/huitzlopochtli/YTMusicUltimate/actions/runs/8848868536/job/24299535693#step:12:46):1: error: import of C++ module 'MachO.nlist' appears within extern "C" language linkage specification [-Wmodule-import-in-extern-c]
#include <mach-o/nlist.h>
^
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:43:1: note: extern "C" language linkage specification begins here
extern "C" {
^
1 error generated.
make[3]: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/instance/rules.mk:387: /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/PremiumStatus.xm.10b1eca7.o] Error 1
In file included from Source/PlaybackRate.xm:20:
==> Compiling Source/PlaybackRate.xm (arm64)…
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:45:1: error: import of C++ module 'MachO.nlist' appears within extern "C" language linkage specification [-Wmodule-import-in-extern-c]
#include <mach-o/nlist.h>
^
/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/vendor/include/substrate.h:43:1: note: extern "C" language linkage specification begins here
extern "C" {
^
rm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/SelectableLyrics.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/NavBar.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/RemoveAds.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/SponsorBlock.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Sideloading.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/OtherSettings.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/AVSwitching.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/DisableAutoPlay.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/VolumeBar/VolumeBar.xm.mm /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/
1 error generated.
make[3]: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/instance/rules.mk:387: /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/Source/PlaybackRate.xm.10b1eca7.o] Error 1
make[2]: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/instance/library.mk:52: /Users/runner/work/YTMusicUltimate/YTMusicUltimate/main/.theos/obj/debug/arm64/YTMusicUltimate.dylib] Error 2
make[1]: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/instance/library.mk:37: internal-library-all_] Error 2
make: *** [/Users/runner/work/YTMusicUltimate/YTMusicUltimate/theos/makefiles/master/rules.mk:1[46](https://github.com/huitzlopochtli/YTMusicUltimate/actions/runs/8848868536/job/24299535693#step:12:47): YTMusicUltimate.all.tweak.variables] Error 2
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -19,7 +19,7 @@ concurrency:
 jobs:
   build:
     name: Build YTMusicUltimate
-    runs-on: macos-latest
+    runs-on: macos-12
     permissions:
       contents: write
 

